processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2025-04-08 14:50:45.211768224 -0400
+++ memcpu.hex	2025-04-08 14:50:45.194767575 -0400
@@ -18,7 +18,6 @@
 :040085000010639371
 :0400860000732023C0
 :04008700FFFFFFFF79
-:0401000000000001FA
 :04010200DEADBEEFC1
 :040103000000DEAD6D
 :040105000000BEEF49
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
